pallets/spin/Cargo.toml

```diff
@@ -0,0 +1,53 @@
+[package]
+name = "pallet-spin"
```

Comment:
I also have pallet SPIN in my PR
https://github.com/QuantumFusion-network/qf-solochain/pull/47/files#diff-5ace5ae0b49a071f3059f30b40462e5ee63ca12353632637015cf5ccdfb85acb

Should we clarify namings which pallet does what?

Reply:
Ok, resolved